packages/forms-web-app/src/controllers/examination/email.js

```diff
@@ -2,24 +2,61 @@ const {
 	routesConfig: {
 		examination: {
 			directory: examinationDirectory,
-			pages: { nameMyself, email }
+			pages: { email }
 		}
 	}
 } = require('../../routes/config');
 
+const config = require('../../config');
+const examinationSessionStorage = config?.sessionStorage?.examination;
+
 const pageData = {
-	backLinkUrl: `${examinationDirectory + nameMyself.route}`,
 	id: email.id,
 	pageTitle: email.name,
-	title: email.name
+	title: email.name,
+	hint: "We'll use your email address to confirm we've received your submission. We will not publish your email address."
 };
 
 const getEmail = async (req, res) => {
+	const requestSession = req.session;
+	const userEmail = requestSession?.[examinationSessionStorage.name]?.userEmail;
```

Comment:
looks like 'userEmail' doesn't need to be used until after the conditional '(!requestSession || !currentView || !currentView?.route)'. Would it be better to declare and set it after?

Reply:
cool will fix it